spec/sections/11-geometry_extension.adoc

```diff
@@ -1952,6 +1993,14 @@ geof:aggCentroid (geom: ogc:geomLiteral): ogc:geomLiteral
 
 The function http://www.opengis.net/def/function/geosparql/aggCentroid[`geof:aggCentroid`] calculates the centroid of the set of given geometries.
 
+==== Function: geof:aggCollect
+
+```
+geof:aggCollect (geom: ogc:geomLiteral): ogc:geomLiteral
+```
+
+The function http://www.opengis.net/def/function/geosparql/aggCollect[`geof:aggCollect`] calculates creates a GeometryCollection literal out of a set of geometries.
```

Comment:
Typo: "calculates creates"

Reply:
does `geom: ogc:geomLiteral` indicate that it's an array?

Reply:
It is a set of ogc:geomLiterals in pratctice likely bound to a SPARQL variable like in other functions too.
The content of the ogc:geomLiterals could be arrays, e.g. a WKT GeometryCollection could be considered as an array.
Does that answer your question or would you have a different idea how to mark an array/set?

Reply:
@situx I saw your answer in another issue that one geomLiteral can carry a collection. 
But I think this is different: aggregates work on a resultset of multiple geomLiterals, so the function signature should indicate an array. Eg use `geomLiteral*`

Reply:
My comment is answered. But @mperry455 's comment not answered yet